src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/Field.spec.jsx

```diff
@@ -101,7 +101,7 @@ describe('Field component', () => {
         testI18nBadgeRender(true, [lang1], false);
     });
 
-    it('should add labelHtmlFor to the label and the field should have a defined id attribute', () => {
+    it('should add labelHtmlFor to the label', () => {
```

Comment:
Vrai question, vu que pour un multiple l'association label => input n'est plus possible. Faudrait trouver des balise aria correspondante.

Reply:
Sinon le htmlFor n'est utile que si le field a un ID. Du coup, je pense que le test est a refaire en fonction de la réponse à la question plus haut.

Reply:
A specific task have been added to handle the accessibility of this multiple fields, current commit is already big. And we can split the work